src/cmd/commit.ts

```diff
@@ -26,6 +27,7 @@ const commitAndPush = async (values: Record<string, any>, branch: string): Promi
   const d = terminal(`cmd:${cmdName}:commitAndPush`)
   d.info('Committing values')
   const argv = getParsedArgs()
+  const forceCommit = isCi && existsSync(`${env.ENV_DIR}/.rerun`)
```

Comment:
The forceCommit name could be interpreted as pushing with --force flag.
Consider followUpCommit or requiresAdditionalCommit